packages/frameworks/src/frameworks.ts

```diff
@@ -1978,6 +1978,7 @@ export const frameworks = [
       },
       outputDirectory: {
         placeholder: 'dist',
```

Comment:
Nice find!

You can remove `placeholder`, its either `placeholder` or `value` but not both